salt/minion.py

```diff
@@ -2152,6 +2157,9 @@ def _refresh_grains_watcher(self, refresh_interval_in_minutes):
             })
 
     def _fire_master_minion_start(self):
+        include_grains = False
+        if self.opts['start_event_grains']:
+            include_grains = True
```

Comment:
Actually just this:
```python
include_grains = bool(self.opts["start_event_grains"])
```
But this is not even needed, same as the whole new parameter to `_fire_master` is redundant and unnecessary.